src/agent/onefuzz-agent/src/local/common.rs

```diff
@@ -26,17 +26,17 @@ pub const CHECK_FUZZER_HELP: &str = "check_fuzzer_help";
 pub const TARGET_EXE: &str = "target_exe";
 pub const TARGET_ENV: &str = "target_env";
 pub const TARGET_OPTIONS: &str = "target_options";
-pub const SUPERVISOR_EXE: &str = "supervisor_exe";
-pub const SUPERVISOR_ENV: &str = "supervisor_env";
-pub const SUPERVISOR_OPTIONS: &str = "supervisor_options";
+// pub const SUPERVISOR_EXE: &str = "supervisor_exe";
```

Comment:
Assuming this is unused but intentionally kept for later?

Reply:
Yes.